plugins/source/gitlab/docs/_configuration.md

```diff
@@ -11,8 +11,8 @@ spec:
 
   # Gitlab specific configuration
   spec:
-    access_token: "<YOUR_ACCESS_TOKEN_HERE>"
-    base_url: "<INSTANCE_URL>" # optional, leave empty for GitLab SaaS
-    # Optional parameters
-    # concurrency: 10000
+    # required
+    access_token: "${GITLAB_ACCESS_TOKEN}"
+    # optional, leave empty for GitLab SaaS
+    # base_url: "<INSTANCE_URL>"
```

Comment:
```suggestion
    # base_url: "${GITLAB_INSTANCE_URL}"
```

Reply:
Actually we won't want since this is a comment (it will show up in the managed sync wizard as a default secret to configure). Also I'm not sure it should be an default env variable, see:

![image](https://github.com/cloudquery/cloudquery/assets/26760571/6f390e6d-94c3-43a5-91ec-f3f82c5c0c8a)